package.json

```diff
@@ -65,6 +65,7 @@
   "dependencies": {
     "async": "^3.1.0",
     "basic-auth": "^2.0.1",
+    "bcrypt": "^5.0.1",
```

Comment:
should we consider "https://github.com/dcodeIO/bcrypt.js" instead to avoid a node-gyp module?

Reply:
I don't know enough about the limitations of node-gyp modules, I assume it's what allows the C++ library to be used here? What would be the disadvantages?

Bcrypt's package site https://www.npmjs.com/package/bcrypt has a warning that versions <5.0 are vulnerable to both bcrypt wrap-around and not properly handling NUL characters inside. bcrypt.js seems to not have had a release since 2017 and probably is not updated to handle these?

Reply:
I think we should address this outside this PR. We need to decide before we perform the v4 release at it will set requirements for hosting Unleash (node-gyp needs to be installed).